es-vue-base/src/lib-components/EsSlider.vue

```diff
@@ -17,15 +17,15 @@
             <!-- Tooltip above the slider thumb -->
             <template #tooltip="{ value, focus }">
                 <div
-                    class="slider-tooltip d-flex h5 align-items-center text-white justify-content-center m-0 bg-primary"
+                    class="slider-tooltip d-flex h4 align-items-center text-white justify-content-center m-0 bg-blue-600 font-weight-boldest"
```

Comment:
Interesting that this was an `h5` before - I believe `h4`, though, makes it larger than the design on desktop (if you drag the slider all the way to the right, the "$1000" doesn't fit inside the circle and the font size is 20px rather than the 16px in the design) - maybe we just remove the class altogether and let it take the default 16px body font size on both mobile and desktop?